### ISSUE TITLE
Is this BatchNorm operation in SGE-Block?

### ISSUE THREAD

**Issue by AmigoCDT** [implus/PytorchInsight#30]
In [SGENet](https://github.com/implus/PytorchInsight/blob/master/classification/models/imagenet/resnet_sge.py#L20)

```
t = xn.view(b * self.groups, -1)
t = t - t.mean(dim=1, keepdim=True)
std = t.std(dim=1, keepdim=True) + 1e-5
t = t / std
t = t.view(b, self.groups, h, w)
t = t * self.weight + self.bias
```

Is this code euqal to BatchNorm2d operation?
Thanks.

**Comment by konioyxgq** [comment]
I have the same problem.
please!

**Comment by konioyxgq** [comment]
@AmigoCDT
